CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 * Integrate jointplot into pairplot, add point-estimate and overlay of plot kinds (#1079)
 * New grayscale style. This also add two new cmaps `cet_grey_r` and `cet_grey_r`. These are perceptually uniform gray scale cmaps from colorcet (linear_grey_10_95_c0) (#1164)
 * Add warmup groups to InferenceData objects, initial support for PyStan (#1126) and PyMC3 (#1171)
+* `hdi_prob` will not plot hpd if argument `hide` is passed. Previously `credible_interval` would omit HPD if `None` was passed  (#1176)
```

Comment:
What do we do for the rest of the hpd mentions?

Reply:
I need to keep going through and fixing it. I got tired yesterday but will start again today